torch_em/transform/generic.py

```diff
@@ -129,11 +134,18 @@ def _get_preprocess_shape(self, oldh, oldw):
         newh = int(newh + 0.5)
         return (newh, neww)
 
+    def _convert_transformed_inputs_to_original_shape(self, resized_inputs):
```

Comment:
I would remove the `_` from the name, since this is an externally facing function.